src/main/python/ttconv/scc/reader.py

```diff
@@ -481,7 +481,8 @@ def process_line(self, line: SccLine) -> SmpteTimeCode:
 
     for scc_word in line.scc_words:
 
-      if self.previous_code == scc_word.value:
+      if self.previous_code == scc_word.value and self.previous_code_type is not str:
```

Comment:
I am thinking we should add a `isControlCode()` in `SccWord`, which would test whether the MSB is a control character.

Reply:
@humberthardy Should I propose an update to your branch?